cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -2522,6 +2481,88 @@
   </property>
 
   <property>
+    <name>system.notification.retry.policy.base.delay.ms</name>
+    <value>100</value>
+    <description>
+      The base delay between retries in milliseconds
+    </description>
+  </property>
+
+  <property>
+    <name>system.notification.retry.policy.max.delay.ms</name>
+    <value>5000</value>
+    <description>
+      The maximum delay between retries in milliseconds
+    </description>
+  </property>
+
+  <property>
+    <name>system.notification.retry.policy.max.retries</name>
+    <value>5000</value>
```

Comment:
The `RetryStrategies` requires it. It is high because in most situation, we don't want it to give up.